commands/create.go

```diff
@@ -351,11 +378,6 @@ func getDriverOpts(c CommandLine, mcnflags []mcnflag.Flag) drivers.DriverOptions
 
 	for _, f := range mcnflags {
 		driverOpts.Values[f.String()] = f.Default()
-
-		// Hardcoded logic for boolean... :(
-		if f.Default() == nil {
-			driverOpts.Values[f.String()] = false
-		}
```

Comment:
I see below you swapped nil for False on Default. Do you know if there was reason it was nil in first place ?

Reply:
The default for booleans is `false`: https://github.com/rancher/machine/blob/master/libmachine/mcnflag/flag.go#L69-L71

Reply:
Yes but you changed it from nil to false, when false is the default by golang, so why did we make the non-standard nil in first place ? @luthermonson ?

Reply:
The logic was if the default was `nil`, then set it to `false`. The link I posted shows that the default was already `false` so there is no need for the check.

Reply:
I'm sorry, I misunderstood which part you were asking about. According to Goland, this is the only place that `BoolFlag.Default()` is used. Not sure why it was set to `nil` originally, but that was from the docker-machine fork.